utility/synapse-utilities-common/src/main/java/io/americanexpress/synapse/utilities/common/email/EmailAddressValidator.java

```diff
@@ -0,0 +1,36 @@
+package io.americanexpress.synapse.utilities.common.email;
+
+import org.apache.commons.lang3.StringUtils;
+import java.util.regex.Pattern;
+
+/**
+ * {@code EmailAddressValidator} is used for email address validation.
+ */
+public class EmailAddressValidator {
```

Comment:
could we just have this class implement ConstraintValidator to consolidate into one class? It doesn't seem like we need two classes

Reply:
I believe these two classes are fine. If the end user would want to use the validator without the annotation it is possible. The ConstrainValidator should be used for the annotation class. This opens for both paths you can use for validating email:

@EmailAddress
private String email;

&

public boolean isEmailValid(String email) {
       return EmailValidator.isValid(email);
}